### PR TITLE
Quarkus 1.11.0.Final

### PR DESCRIPTION
**Pull request by gsmet**

### CODE REVIEW

File: _posts/2021-01-20-quarkus-1-11-0-final-released.adoc [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+
+=== Jackson
+
+`FAIL_ON_UNKNOWN_PROPERTIES` is now disabled by default for the default CDI-enabled `ObjectMapper`.
+
+See https://quarkus.io/guides/rest-json#jackson for more details about how to go back to the previous behavior.
+
+=== jbang dev mode
+
+The integration with https://jbang.dev[jbang] now supports enabling development mode.
```
Comment (maxandersen) [inline _posts/2021-01-20-quarkus-1-11-0-final-released.adoc:116]:
```suggestion
The integration with https://jbang.dev[jbang] now supports enabling development mode and can use Quarkus platforms (BOM) for version management.
```

File: _posts/2021-01-20-quarkus-1-11-0-final-released.adoc [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+Until now, it was only possible to inject the JPA `EntityManagerFactory` and `EntityManager` in your applications.
+Starting with 1.11, you can inject the Hibernate ORM-specific `SessionFactory` and `Session` if you need to access their specific methods. 
+
+=== Jackson
+
+`FAIL_ON_UNKNOWN_PROPERTIES` is now disabled by default for the default CDI-enabled `ObjectMapper`.
+
+See https://quarkus.io/guides/rest-json#jackson for more details about how to go back to the previous behavior.
+
+=== jbang dev mode
```
Comment (maxandersen) [inline _posts/2021-01-20-quarkus-1-11-0-final-released.adoc:114]:
```suggestion
=== jbang dev mode & platform support
```

File: _posts/2021-01-20-quarkus-1-11-0-final-released.adoc [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+
+The integration with https://jbang.dev[jbang] now supports enabling development mode.
+For this you need to use jbang v0.61+.
```
Comment (maxandersen) [inline _posts/2021-01-20-quarkus-1-11-0-final-released.adoc:117]:
```suggestion
For this you need to use jbang v0.62+.
```

File: _posts/2021-01-20-quarkus-1-11-0-final-released.adoc [hunk 4]
```diff
@@ -0,0 +1,193 @@
...
+jbang -Dquarkus.dev myapp.java
+----
+
+where `myapp.java` for example is the following:
+
+[source, java]
+----
+//usr/bin/env jbang "$0" "$@" ; exit $?
+//DEPS io.quarkus:quarkus-resteasy:1.11.0.Final
+//DEPS io.quarkus:quarkus-smallrye-openapi:1.11.0.Final
```
Comment (maxandersen) [inline _posts/2021-01-20-quarkus-1-11-0-final-released.adoc:132]:
```suggestion
//DEPS io.quarkus:quarkus-bom:1.11.0.Final@pom
//DEPS io.quarkus:quarkus-resteasy
//DEPS io.quarkus:quarkus-smallrye-openapi
```

File: _posts/2021-01-20-quarkus-1-11-0-final-released.adoc [hunk 5]
```diff
@@ -0,0 +1,193 @@
...
+    }
+
+    @GET
+    public String sayHello() {
+        return "hello from Quarkus with jbang.dev";
+    }
+}
+----
+
+Then you can edit that file and save it again and Quarkus live reload feature will take care of applying the changes.
```
Comment (maxandersen) [inline _posts/2021-01-20-quarkus-1-11-0-final-released.adoc:157]:
```suggestion
Notice the use of `@pom` in the first line; that instructs jbang to load the pom for for platform and use it for version management and the other dependency versions can be left out. 

Then with dev mode you can edit that file and save it again and Quarkus live reload feature will take care of applying the changes.
```

**Review by maxandersen — DISMISSED** [review]
added support for bom in jbang so might as well mention it for quarkus release.

**Comment by rsvoboda** [comment]
I was looking for render of 1.11 announce in https://quarkusio-quarkusio-github-io-preview-pr-831.surge.sh/blog/tag/release/ but without luck. Where can I find the html render for the planned release?

Also noticed that release announces uses just `#release` tag. I was initially looking at `#announcement` tag - https://quarkusio-quarkusio-github-io-preview-pr-831.surge.sh/blog/tag/announcement/.
Is `#announcement` tag reserved for non-release announcements?

**Comment by gsmet** [comment]
Yeah that's because the post is in the future.

As for the tag, I have no idea. We have been using these tags for ages.

**Comment by rsvoboda** [comment]
Thanks, missed `2021-01-20` part. About tag, that was just an idea, if nobody else complained about it it can stay as is now, just clicking on announce was easy because it is at the top :)

**Comment by phillip-kruger** [comment]
@gsmet @kenfinnigan - should there not something about the non-application path (`/q`) ? It's new in this release ? It might cause many questions if we have nothing about it ?

**Comment by gsmet** [comment]
@phillip-kruger there is something about it :)

And it's also in the migration guide.

**Comment by phillip-kruger** [comment]
@gsmet - apologies , I missed it (reading in GH review :) ) Great ! Thanks .

**Pull request merged**